### PR TITLE
workaround for spark1 script

### PR DESCRIPTION
**Pull request by yaojiefeng**
For spark1 on dataproc, the script sometimes returns empty result, which makes the pipeline super flaky. Adding a sleep will make the script always return results(at least from my 50+ testing)

### CODE REVIEW

**Review by CuriousVini — APPROVED** [review]

File: cdap-app-fabric/src/main/resources/setupSpark1.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# A simple python that runs with spark-submit to grab all SPARK environment variables
+import os, time
+for key, val in os.environ.items():
+    if key.startswith("SPARK_") or key.startswith("HADOOP_"):
+        print("export {}=\"{}\"".format(key, val))
+time.sleep(1)
```
Comment (chtyim) [inline cdap-app-fabric/src/main/resources/setupSpark1.py:23]: Add a comment of why the sleep is needed.

**Review by chtyim — APPROVED** [review]

File: cdap-app-fabric/src/main/resources/setupSpark1.py (hunk 1 above)
Comment (yaojiefeng) [reply cdap-app-fabric/src/main/resources/setupSpark1.py:23]: Added

**Comment by yaojiefeng** [comment]
close this since we have better fix

**Pull request closed without merging**